ml.prevete.Daty: 1.0alpha release

tallero opened the pull request:
Can you help me tag the unnecessary finish-args? At the moment the app would just need display windows, read data into its XDG directories and connect to internet.

bilelmoussaoui commented on org.prevete.Daty.json:
```diff
@@ -0,0 +1,59 @@
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "sdk" : "org.gnome.Sdk",
+    "tags" : [
```
Not needed.

bilelmoussaoui commented on org.prevete.Daty.json:
```diff
@@ -0,0 +1,59 @@
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://source.puri.sm/Librem5/libhandy.git"
```
You should be building a stable release.

bilelmoussaoui commented:
testbot, build org.prevete.Daty

FakeShemp commented:
Is prevete.org down or is it a temporary URL?

bilelmoussaoui commented on org.prevete.Daty.json:
```diff
@@ -0,0 +1,59 @@
+            "name" : "libhandy",
+            "buildsystem" : "meson",
+            "config-opts": [
+                "-Dglade_catalog=disabled"
```
You should disable examples, tests, vapi too here.

bilelmoussaoui replied:
vapi is enabled by default and you don't need that for a python app.

tallero commented:
@FakeShemp temporary

bilelmoussaoui commented on org.prevete.Daty.json:
```diff
@@ -1,11 +1,5 @@
 {
-    "app-id": "org.prevete.Daty",
-    "runtime" : "org.gnome.Platform",
```
You still need the runtime, the runtime-version, the sdk too. it's just the tags that I meant;

tallero replied:
Sorry, I have not found documentation on tags, so I thought of them of generic tags.

bilelmoussaoui replied:
Tags are not used here on Flathub at all unless you want to tag the software as proprietary.

nedrichards commented:
bot, build ml.prevete.Daty
testbot, build ml.prevete.Daty

nedrichards commented:
@tallero this build will fail - your app-id and the name of the json file need to be the same (it's a flathub thing).

nedrichards commented:
bot, build ml.prevete.Daty
testbot, build ml.prevete.Daty

tallero commented:
@barthalion It took time to do nice screenshots and an overview picture for the repo. Apparently I can't format the description nor links, am I right?

barthalion commented on appstream.xml:
```diff
@@ -0,0 +1,48 @@
+            </description>
+        </release>
+    </releases>
+</component>
```
Also needs OARS rating: https://hughsie.github.io/oars/

Ideally it should be shipped with your code so other distributions can also use it but it's okay for now.

barthalion commented:
testbot, build ml.prevete.Daty

nedrichards commented:
testbot, build ml.prevete.Daty
bot, build ml.prevete.Daty

nedrichards commented:
testbot, build ml.prevete.Daty
bot, build ml.prevete.Daty

nedrichards commented:
@tallero you should ideally put the appdata into your project upstream and install it there as part of a new release - so it stays up to date and is part of your project.

In this case I just quickly fixed it so the file was declared correctly within the manifest and installed from there.

nedrichards commented on ml.prevete.Daty.appdata.xml:
```diff
@@ -0,0 +1,58 @@
+  <developer_name>Pellegrino Prevete</developer_name>
+
+  <!--FIXME: where to submit translations-->
+  <url type="translate"><!-- http://www.homepage.com/how-to-submit-translations.html --></url>
```
you should remove this if it's not available.

nedrichards commented on ml.prevete.Daty.appdata.xml:
```diff
@@ -0,0 +1,58 @@
+  <project_license>GPL-3.0</project_license>
+  <description>
+    <p>
+      <b>Daty is a Wikidata editor</b>
```
Something a bit better here would be good. How about this (lightly adapted from the README)
> Daty is a free Wikidata editor adhering to GNOME Human Interface Guidelines, intended to enable a better editing workflow and faster deployment of requested user features for Wikidata.
Use Daty to search, select, read, batch edit items, script actions, share, visualize proposed changes and bots.

tallero replied:
Thank you!

nedrichards commented:
Builds fine now: https://flathub.org/builds/#/builders/2/builds/7726

nedrichards commented:
testbot, build ml.prevete.Daty
bot, build ml.prevete.Daty

barthalion commented:
It does not start for me:

```

tallero commented:
@barthalion I noticed that.
Apparently I forgot to change the reverse-dns in the user setup when fixing the domain for the release. I have fixed that and moved the version tag to actual HEAD, so recompiling now should produce a good build.
I sincerely hope this is the last test build.

nedrichards commented:
@tallero cool! when you push an update here i'll kick off a build

tallero commented:
@nedrichards it is working, you can test!

barthalion commented:
testbot, build ml.prevete.Daty
bot, build ml.prevete.Daty

barthalion commented:
Repository has been created: https://github.com/flathub/ml.prevete.Daty

Thanks!

tallero commented:
@barthalion Should I have the permission to push changes to that repository?

barthalion commented:
Yes, although Github is a bit confusing about it. I invited you to collaborators; you should have received the invitation. It has to be accepted before you're granted write access.

barthalion commented:
According to documentation, https://github.com/flathub/ml.prevete.Daty/invitations should let you accept it.
